packages/dev/core/src/Cameras/targetCamera.ts

```diff
@@ -370,7 +361,14 @@ export class TargetCamera extends Camera {
 
         // Move
         if (needToMove) {
-            this._updatePosition();
+            // If no inertia, use full value and then zero out cameraDirection
+            if (this.inertia === 0) {
+                this._updatePosition(relativeInertia, scaleFactor);
+                this.cameraDirection.scaleInPlace(0);
+            } else {
+                this.cameraDirection.scaleInPlace(relativeInertia);
+                this._updatePosition(relativeInertia, scaleFactor);
+            }

```

Comment:
that's technically the only change that was needed to go back to the way things were before